packages/dx-react-grid/src/plugins/grouping-panel.jsx

```diff
@@ -13,7 +13,13 @@ export class GroupingPanel extends React.PureComponent {
     } = this.props;
 
     return (
-      <PluginContainer>
+      <PluginContainer
+        pluginName="GroupingPanel"
+        dependencies={[
+          { pluginName: 'GroupingState' },
+          { pluginName: 'SortingState', optional: true },
```

Comment:
We already have all information about options that turns dependency to required. Why is it not reflected here?